stylus/ui-app/alerts.styl

```diff
@@ -19,7 +19,14 @@ $alerts
         left       0
         background white
         opacity    .6
-        transition opacity ease-out .3s
+        visibility visible
```

Comment:
It was quite complicated to animate the overlay here because when you just put the opacity to 0, it's still 'clickable' et avoid the user to click on other parts of the app. As it's not possible to use transition with `display:none`, the `visibility` allows to disable the 'clickable behaviour' by being compatible with transition (by using a delay of 0.3s for the hiding transition)